src/main/java/net/minecraftforge/fluids/FluidType.java

```diff
@@ -219,6 +225,17 @@ public Rarity getRarity()
         return this.rarity;
     }
 
+    /**
+     * Returns the Pointed Dripstone drip information of the fluid.
+     * <p> Note that this CAN be null!
```

Comment:
What does null mean then? Also, nullable can be assumed from the method signature.